moto/redshift/models.py

```diff
@@ -749,6 +751,16 @@ def delete_cluster_security_group(self, security_group_identifier):
             return self.security_groups.pop(security_group_identifier)
         raise ClusterSecurityGroupNotFoundError(security_group_identifier)
 
+    def authorize_cluster_security_group_ingress(self, security_group_name, cidr_ip):
+        security_group = self.security_groups[security_group_name]
+        if not security_group:
+            raise ClusterSecurityGroupNotFoundFaultError("")
```

Comment:
Can we get a unit test for this scenario? I'm guessing the message should have a value as well - would be good to assert on that